kroxylicious-app/pom.xml

```diff
@@ -175,6 +175,10 @@
     <profiles>
         <profile>
             <id>dist</id>
+            <properties>
+                <io.kroxylicious.proxy.image.name>quay.io/kroxylicious/proxy:${project.version}</io.kroxylicious.proxy.image.name>
+                <io.kroxylicious.proxy.image.archive>target/kroxylicious-proxy.img.tar.gz</io.kroxylicious.proxy.image.archive>
```

Comment:
Doing this via system properties makes things quite clunky to setup in IntelliJ.

Reply:
Yeah, but env vars (if that's what you're angling for?) are only marginally better.

Reply:
I'm not angling for anything in particular. As discussed else where default so things just work would be ideal.